src/app/src/components/Navbar/GoogleTranslateButton.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+
+export default () => (
+    <div
+        style={{ position: 'relative' }}
+        className="google-translate-container"
+    >
+        <div
+            id="google_translate_element"
+            style={{
+                position: 'relative',
+                opacity: 0,
+                zIndex: 4,
+                right: 0,
+            }}
+        />
+        <div
+            style={{
+                position: 'absolute',
+                left: 0,
+                top: 0,
+                zIndex: 3,
+                display: 'flex',
+                height: '100%',
+                alignItems: 'center',
+            }}
+        >
+            <Button
+                type="button"
+                className="nav__parent-button app-header-button google-translate-language-display"
+                style={{
+                    backgroundColor: 'white',
+                    color: '#0c46e1',
+                }}
+            >
+                <span
+                    className="skiptranslate"
```

Comment:
"skiptranslate" here is extremely important. Without this class, Google translate attempts to translate this span, replacing the inner text (language code) with a <font> element with a translation of the current language code from the previously selected language into the current language in cases where the code is a word in the previously selected language. This causes extremely buggy and hard-to-track behavior, such as 'EN' being replaced by 'ON' or 'OF' depending on the 'source' language. @lederer This is the source of the strange behavior we saw yesterday.